### PR TITLE
ROC-2556: Enable collection of statement of means

### PR DESCRIPTION
**Pull request by rlewan**
### JIRA link ###

https://tools.hmcts.net/jira/browse/ROC-2556

### Change description ###

This PR adds the start page for statement of means functionality and sets the layout for future SOM pages.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/features/statement-of-means/paths.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { RoutablePath } from 'common/router/routablePath'
+
+const statementOfMeansPath = '/case/:externalId/statement-of-means'
+
+export class Paths {
+  static readonly startPage = new RoutablePath(`${statementOfMeansPath}/start`)
```
Comment (timja-kainos) [inline src/main/features/statement-of-means/paths.ts]: shouldn't this be under the response journey?

File: src/main/features/statement-of-means/views/start.njk [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{% extends "layout.njk" %}
+
+{% set title = t('Money claim') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-large">You need to answer some financial questions</h1>
```
Comment (timja-kainos) [inline src/main/features/statement-of-means/views/start.njk]: translating..

File: src/main/features/statement-of-means/views/start.njk [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{% extends "layout.njk" %}
+
+{% set title = t('Money claim') %}
```
Comment (timja-kainos) [inline src/main/features/statement-of-means/views/start.njk]: no need to translate

File: src/main/features/statement-of-means/paths.ts (hunk 1 above)
Comment (rlewan) [reply src/main/features/statement-of-means/paths.ts]: Yeah, probably makes sense to add it there.

Comment (rlewan) [reply src/main/features/statement-of-means/paths.ts]: Doesn't work too nicely with features though :(

**Pull request merged**